# goal_pose_offset (unit: m)
front_cam_big_tag_offset_x: 0.1
front_cam_big_tag_offset_y: 0.5
front_cam_small_tag_offset_x: 0.0
front_cam_small_tag_offset_y: 0.0
rear_cam_small_tag_offset_x: 4.0
rear_cam_small_tag_offset_y: 0.0

# frame_id
base_frame: base_link

# tag_detectioin topic
tag_detect_info_topic_name: tag_detections
rear_camera_tag_detect_info_topic_name: rear_/tag_detections

# approach threshold
approach_dist_threshold: 0.2 # meter
approach_time: 25 # 2.5 sec